api/room/schema.py

```diff
@@ -191,13 +230,17 @@ def mutate(self, info, room_id, **kwargs):
 
 
 class UpdateFirebaseToken(graphene.Mutation):
+    """
+        Class to update the firebase token
+    """
 
     class Arguments:
```

Comment:
this is not documented
<img width="375" alt="screenshot 2019-02-28 at 18 00 05" src="https://user-images.githubusercontent.com/23398223/53575567-e24d1300-3b82-11e9-8b9f-9c79b4206437.png">